### PR TITLE
[TRANSLATION] translate CS144.md

### PR DESCRIPTION
**Pull request by clingfei**

### CODE REVIEW

File: docs/计算机网络/CS144.en.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Introduction
+
+- Offered by: Stanford
+- Prerequisites: Computer System Fundamentals, CS106L
+- Programming Language: C++
+- Difficulty: 🌟🌟🌟🌟🌟
+- Class Hour: 100 hours
+
+One of the lecturers of this course is Professor [Nick McKeown](http://yuba.stanford.edu/~nickm/index.html), a giant in the field of Networking. At the end of each chapter of MOOC, he will interview an executive in the industry or an expert in the academia, which can certainly broadens your horizons.
```
Comment (PKUFlyingPig) [inline docs/计算机网络/CS144.en.md:11]: which can certainly broaden your horizons

File: docs/计算机网络/CS144.en.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+In the projects, you will use C++ to build the entire TCP/IP protocol step by step, implement IP router and ARP protocol, and finally use your own protocol stack to replace the protocol stack in Linux Kernel to communicate with your classmates, which is really amazing and exciting.
```
Comment (PKUFlyingPig) [inline docs/计算机网络/CS144.en.md:13]: you will use C++ to build the entire TCP/IP protocol stack, the IP router, and the ARP protocol step by step from scratch. Finally, you will replace Linux Kernel's protocol stack with your own and use socket programming to communicate with your classmates, which is really amazing and exciting.

File: docs/计算机网络/CS144.en.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Resources
+
+- Website: https://cs144.github.io/
```
Comment (PKUFlyingPig) [inline docs/计算机网络/CS144.en.md:17]: Course Website: `<https://cs144.github.io/>`

File: docs/计算机网络/CS144.en.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+- Video: https://www.youtube.com/watch?v=r2WZNaFyrbQ&list=PL6RdenZrxrw9inR-IJv-erlOKRHjymxMN
```
Comment (PKUFlyingPig) [inline docs/计算机网络/CS144.en.md:18]: Video: `<https://www.youtube.com/watch?v=r2WZNaFyrbQ&list=PL6RdenZrxrw9inR-IJv-erlOKRHjymxMN>`

File: docs/计算机网络/CS144.en.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+- Textbook: None
+- Assignments: <https://cs144.github.io/>
```
Comment (PKUFlyingPig) [inline docs/计算机网络/CS144.en.md:20]: Assignments: refer to the course website

**Comment by clingfei** [comment]
Thanks for your review, I have pushed a new commit.

**Pull request merged**